kornia/color/_colormap_data.py

```diff
@@ -0,0 +1,1263 @@
+from typing import List
+
+# All colormaps has 64 base colors. Extracted from matplotlib.
+# https://matplotlib.org/stable/users/explain/colors/colormaps.html
+
+
+RGBColor = List[float]
+
+
+_AUTUMN_BASE: List[RGBColor] = [
+    [1.0, 0.0, 0.0],
+    [1.0, 0.015873015873015872, 0.0],
+    [1.0, 0.031746031746031744, 0.0],
+    [1.0, 0.047619047619047616, 0.0],
+    [1.0, 0.06349206349206349, 0.0],
+    [1.0, 0.07936507936507936, 0.0],
+    [1.0, 0.09523809523809523, 0.0],
+    [1.0, 0.1111111111111111, 0.0],
```

Comment:
can you wrap these base values under functions? something like 
```suggestion
def get_autumn_base() -> List[RGBColor]:
    return [
    [1.0, 0.0, 0.0],
    [1.0, 0.015873015873015872, 0.0],
    [1.0, 0.031746031746031744, 0.0],
    [1.0, 0.047619047619047616, 0.0],
    [1.0, 0.06349206349206349, 0.0],
    [1.0, 0.07936507936507936, 0.0],
    [1.0, 0.09523809523809523, 0.0],
    [1.0, 0.1111111111111111, 0.0],
```

Reply:
Yes, i forget to do this.  If I add a function in CMAP (enum), I will have to call these functions, right?

```
class CMAP(Enum):
    autumn = 1
    bone = 2
    jet = 3
    def _load_base(self) -> RGBColor:
        return {
            "autumn": get_autum_base(),
            "bone": get_bone_base(),
            "jet": get_jet_base(),
        }[self.name]
```

Reply:
yes, but calling function call can be on return 

```python
class CMAP(Enum):
    autumn = 1
    bone = 2
    jet = 3
    def _load_base(self) -> RGBColor:
        return {
            "autumn": get_autum_base,
            "bone": get_bone_base,
            "jet": get_jet_base,
        }[self.name]()
```